app/views/to_do_items/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_for @item = list.items.new,
```

Comment:
@vlasiak, Don't you think that is a responsibility of a new action to create a new resource (object)? It means that once a user wants to create an item `new` action is being triggered.